### PR TITLE
Adds legal roles

### PR DESCRIPTION
**Pull request by gabrielamatias**
@jacobpgn , @jesssummerfield : please review.

### CODE REVIEW

**Comment by jacobpgn** [comment]
👍

File: app/pages/about/jobs/positions/compliance-manager.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        </p>
+        <ul className='list u-margin-Tm'>
+          <li>You’ve helped to establish, or have seen a compliance function established.</li>
+          <li>You have knowledge of FCA requirements.</li>
+          <li>You have experience of submitting updates to the FCA and/or HMRC.</li>
+          <li>You have experience of an international compliance programme.</li>
+        </ul>
+        <p className='para'>
+          This role would ideally suit someone with experience in a large compliance team, who is looking to implement a compliance
+          function at a younger, high-performing company.  However, we recognise that the most talented people can have the most
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/compliance-manager.js]: @gabrielamatias teeny thing but are there two spaces after the full stop and before however? Might just be my eyes...

File: app/pages/about/jobs/positions/legal-assistant.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          You will work closely with our Head of Legal and external advisors to provide the business with the support it needs to grow.
+        </p>
+        <p className='para'>
+          Some of the main elements of the role will involve:
+        </p>
+        <ul className='list'>
+          <li>Advising and leading on a range of sales contract issues and negotiations &ndash; from simplified online terms,
+          through to complex agreements with some of the world’s largest brands.</li>
+          <li>Supporting large legal projects such as venture capital fund raises, and other corporate work.</li>
+          <li>Implementing innovative solutions to and advising on escalated customer support issues, including official complaints.</li>
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/legal-assistant.js]: @gabrielamatias on second thoughts, maybe remove this one if Ahmed's ok with it? Makes it look like we have loads of issues!

**Comment by jesssummerfield** [comment]
@gabrielamatias all good apart from the two teeny things!

**Pull request merged**